baggage/baggage.go

```diff
@@ -392,7 +392,7 @@ func New(members ...Member) (Baggage, error) {
 		}
 	}
 
-	// Check member numbers after deduplicating.
```

Comment:
Deduplication is the correct and intended term here.

Reply:
Got it! will be repaired soon.